Suggested fix for issue #3

mwittig opened the pull request:
https://github.com/fraunhoferfokus/peer-upnp/issues/3 - The case where url.pathname does not start with peer.prefix is not handled, i.e. no response is generated for this case.

mwittig committed:
Suggested fix for issue https://github.com/fraunhoferfokus/peer-upnp/issues/3 - The case where url.pathname does not start with peer.prefix is not handled, i.e. no response is generated for this case.

mwittig committed:
Suggested fix for issue https://github.com/fraunhoferfokus/peer-upnp/issues/3 - The case where url.pathname does not start with peer.prefix is not handled, i.e. no response is generated for this case.

mwittig committed:
Suggested implementation for option 2 described in https://github.com/fraunhoferfokus/peer-upnp/issues/2

mwittig committed:
Suggested implementation for option 2 described in https://github.com/fraunhoferfokus/peer-upnp/issues/2

louaybassbouss commented:
As I mentioned in my comment on issue #2 , there is no need for the changes you made in this PR. It is important to know that peer-upnp is only responsible for requests starting with "/upnp". All other requests need to be handled in your application (e.g. upnp-test.js). This allows you to reuse the server for other purposes and not only for peer-upnp. I hope the example makes my explanation more clear. If this solves you issue I will reject your PR and close the issue.

mwittig committed:
Revert "Suggested implementation for option 2 described in https://github.com/fraunhoferfokus/peer-upnp/issues/2"

This reverts commit 26443f361ae26c12b8b62444642c63a147034c38.

mwittig committed:
Revert "Suggested implementation for option 2 described in https://github.com/fraunhoferfokus/peer-upnp/issues/2"

This reverts commit 26443f361ae26c12b8b62444642c63a147034c38.

mwittig committed:
Revert "Suggested fix for issue https://github.com/fraunhoferfokus/peer-upnp/issues/3 - The case where url.pathname does not start with peer.prefix is not handled, i.e. no response is generated for this case."

This reverts commit a7e781947e2535f96ed7901916c2af22e267dfd8.

mwittig committed:
Revert "Suggested fix for issue https://github.com/fraunhoferfokus/peer-upnp/issues/3 - The case where url.pathname does not start with peer.prefix is not handled, i.e. no response is generated for this case."

This reverts commit a7e781947e2535f96ed7901916c2af22e267dfd8.

mwittig committed:
Revert "Revert "Suggested fix for issue https://github.com/fraunhoferfokus/peer-upnp/issues/3 - The case where url.pathname does not start with peer.prefix is not handled, i.e. no response is generated for this case.""

This reverts commit f821f9cb49838c936a386c3df71efb30c8e92307.

mwittig committed:
Revert "Revert "Suggested fix for issue https://github.com/fraunhoferfokus/peer-upnp/issues/3 - The case where url.pathname does not start with peer.prefix is not handled, i.e. no response is generated for this case.""

This reverts commit f821f9cb49838c936a386c3df71efb30c8e92307.

mwittig committed:
Revert "Revert "Suggested implementation for option 2 described in https://github.com/fraunhoferfokus/peer-upnp/issues/2""

This reverts commit cf8c4f9ad0b7ff0a8270936db9e3f97fc53b1c1a.

mwittig committed:
Revert "Revert "Suggested implementation for option 2 described in https://github.com/fraunhoferfokus/peer-upnp/issues/2""

This reverts commit cf8c4f9ad0b7ff0a8270936db9e3f97fc53b1c1a.

mwittig committed:
Updated dependencies. Now using peer-ssdp fork with suggested fix for peer-ssdp issue #3: addMembership EADDRNOTAVAIL revisited -

mwittig committed:
Updated dependencies. Now using peer-ssdp fork with suggested fix for peer-ssdp issue #3: addMembership EADDRNOTAVAIL revisited -

louaybassbouss committed:
* Update version to 0.0.4

* Changes:
   * Update to peer-ssdp version 0.0.5

Commit:
fixes #9

Commit:
fixes #9

Commit:
fixes #9

Commit:
fixes #9

Commit:
fixes #9

fixed checking of path to identify peer requests

louaybassbouss committed:
Merge pull request #10 from dittodhole/master

fixes #9

louaybassbouss committed:
* Version updated to 0.0.5

aldafu committed:
Fix setting xml namespace for responses

xmlns for response now includes the full namespace with schema, service,
type and version

aldafu committed:
Merge pull request #13 from fraunhoferfokus/fix-response-xmlns

Fix setting xml namespace for responses

mwittig committed:
Removed duplicate line after merge

mwittig committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	lib/peer-upnp.js

mwittig committed:
update ejs version to 2.5.7

(cherry picked from commit 6236fa6886c3c28183d7bb7e39251ae92156db67)
